en/docs/release-notes/version-3.3.0.md

```diff
@@ -21,22 +21,22 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 ## Integrated GraphQL Tryit
 ![graphql-tryit](../../img/release-notes/graphql-tryit.gif)
 
-- You can try out the GraphQL services using the `Tryit` codelens 
+- You can try out the GraphQL services using the `Tryit` CodeLens 
 - You can send and test any request with headers to your GraphQL service in VS Code itself without using any third-party tools
 - The GraphQL explorer will help you to explore the available endpoints and generate the query for you
 
-## Record editor
+## Improved Record Editor
 ![record-editor](../../img/release-notes/record-editor.gif)
 
 - The expression editor is now combined with the record editor, which will give a better editing experience with suggestions and library support
-- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
-- Record creation using JSON samples now supports separate record creation for complex JSON records. 
+- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation
```

Comment:
Why are we removing the full-stop. Recommendation is to have full-stops. https://developers.google.com/style/periods#bulleted-lists

Reply:
Fixed in https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/24.

Reply:
Why sending another PR while this is still open?

Reply:
Sorry, that was my bad.